

In file included from include/asm/io.h:365,
                 from drivers/video/kyro/STG4000Reg.h:23,
                 from drivers/video/kyro/STG4000Ramdac.c:16:
include/asm/eeh.h:58: warning: `struct device_node' declared inside parameter list
include/asm/eeh.h:58: warning: its scope is only this definition or declaration, which is probably not what you want

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-sparc64-akpm/include/asm-ppc64/eeh.h |    1 +
 1 files changed, 1 insertion(+)

diff -puN include/asm-ppc64/eeh.h~ppc64-eeh-warning-fix include/asm-ppc64/eeh.h
--- 25-sparc64/include/asm-ppc64/eeh.h~ppc64-eeh-warning-fix	2004-06-19 23:41:18.412250488 -0700
+++ 25-sparc64-akpm/include/asm-ppc64/eeh.h	2004-06-19 23:41:41.241779872 -0700
@@ -55,6 +55,7 @@ void __init pci_addr_cache_build(void);
  * device (including config space i/o).  Call eeh_add_device_late
  * to finish the eeh setup for this device.
  */
+struct device_node;
 void eeh_add_device_early(struct device_node *);
 void eeh_add_device_late(struct pci_dev *);
 
_
